### ISSUE TITLE
Text in parentheses is grey

### ISSUE THREAD

**Issue by petrbouchal** [notaapp/talk#311]
Not sure if this is intended or a bug related to rendering of e.g. URLs in links:

<img width="587" alt="image" src="https://user-images.githubusercontent.com/1666657/189915862-3bf2dd0a-6c5e-4cb9-b18d-fce6029b9f66.png">

There would be some logic from the writer's point of view to de-emphasising text in parens, but it does feel unexpected.

**Comment by astoilkov** [comment]
Hi. This is intended behavior.

I agree it feels unexpected. I've never seen it anywhere else actually. I think the approach has both benefits and disadvantages.

I'm not sure what's the better styling. I will label this as a discussion if someone wants to join and share their view.

**Comment by qazzym** [comment]
Personally, I love it! It's testament to the thoughtful design and care that is characteristic of Nota so far. This particular styling is one of those things that is so obvious once you see it for the first time, but no-one else has really thought of or implemented before.

**Comment by bellebethcooper** [comment]
I also found it unexpected but I like it. I tend to write long phrases or full sentences in parentheses a lot, and I think they work really well with this styling. I can see that it might seem more strange when you only use one or two words in parentheses in the middle of a sentence.

**Comment by filipgrkinic** [comment]
Have to agree. 

I wouldn't ask for this feature, but now that I have it, I wouldn't want to lose it.

**Comment by gommammi** [comment]
Agreed! This is a lovely feature for me which lets me focus on the main text, rather than side notes. One thing I noticed is that the shade of grayness is stronger in the dark mode. (The gray styling in the dark mode is significantly more gray than in the light mode.) I couldn't even notice that the letters in parentheses were gray in the light mode (especially with lighter weight fonts). 

But I also felt that this is a bit unconventional for a text editor. I'd like to see this feature as optional depending on the users' preference. (Which could be toggled in the preference pane.)

**Comment by cosmincrihan** [comment]
+1 for adding a setting for this behavior.
I found it confusing at first, but then I kind of enjoyed the look of the text with faded parenthesis explanations. It depends on what you are writing though, so an option to toggle it would go a long way!
